spec/helpers/confirmations_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the ConfirmationsHelper. For example:
+#
+# describe ConfirmationsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe ConfirmationsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
And the specs? 🤔

Reply:
I  going to create the spec that is missing

Reply:
In this moment I'm researching how implement Rspec with the helpers.